src/main/scala/com/databricks/labs/overwatch/pipeline/GoldTransforms.scala

```diff
@@ -934,7 +936,7 @@ trait GoldTransforms extends SparkSessionWrapper {
     """
       |organization_id, workspace_name, instance_pool_id, serviceName, timestamp, date, actionName, instance_pool_name, node_type_id,
       |idle_instance_autotermination_minutes, min_idle_instances, max_capacity, preloaded_spark_versions,
-      |azure_attributes, create_details, delete_details, request_details
+      |azure_attributes, create_details, delete_details, request_details,tags
```

Comment:
these are for end-user views -- the order is important, just sticking this at the end may not be the best place. I moved it after the cloud_attributes columns